cluster/gke/util.sh

```diff
@@ -91,6 +91,7 @@ function verify-prereqs() {
     gcloud_prompt="-q"
   fi
   gcloud ${gcloud_prompt:-} components update preview || true
```

Comment:
We still need the preview component for instance-groups and managed-instance-groups, which were not moved to alpha.